data/content_scripts/posting_process/sendkeys.js

```diff
@@ -0,0 +1,766 @@
+// Cross-broswer implementation of text ranges and selections
+// documentation: http://bililite.com/blog/2011/01/17/cross-browser-text-ranges-and-selections/
+// Version: 2.6
+// Copyright (c) 2013 Daniel Wachsstock
+// MIT license:
```

Comment:
If Wish's solution doesn't help, use the whole library for now and we will
solve performance later. Premature optimization is the root of all evil.